### ISSUE TITLE
[OS X?] Use more "persistent" desktop notifications for certain events

### ISSUE THREAD

**Issue by guruz** [owncloud/client#4697]
At the moment all notifications disappear after some seconds.

E.g. if a conflict file was created, we probably want the notification to stay until the user handled it by clicking it.

Also, clicking it should open the local directory.

FYI @dragotin @jancborchardt

**Comment by jancborchardt** [comment]
Yeah, I agree. Change notifications or others just notifying of an action should disappear after some seconds. But everything where user intervention is needed should stay until closed/clicked or acted upon.

**Comment by dragotin** [comment]
This is not doable with the current Qt capabilities across platforms. 

Note that I started to work on a branch that uses the snorenotify https://github.com/KDE/snorenotify
Lets see how this improves the situation.

**Comment by guruz** [comment]
@dragotin Nice...

We already have OS X specific code btw.. https://github.com/owncloud/client/blob/d7bd1300a8397c2782e8d75cf7c595b1ada70d88/src/gui/systray.mm#L25

**Comment by guruz** [comment]
@dragotin Looks like a nice lib. Can we (at least on Mac and Windows) include it in source tree so we don't get into dependency/build issues like we always do with qtkeychain?

**Comment by guruz** [comment]
Related https://github.com/owncloud/client/issues/4410
